testutil/ext_err.go

```diff
@@ -0,0 +1,74 @@
+package testutil
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+const envFlag = "GH_ANNOTATION"
+
+// ExtError is a wrapper to the error type, extending it with the file and line
+// that produced the error, and a generic error description that is sent to stdout
+// as a Github annotation when the envFlag environment variable is set.
+type ExtError struct {
+	err  error
+	file string
+	line int
+	desc string
+}
+
+func newExtError(err error, desc string) ExtError {
+	_, fn, ln, ok := runtime.Caller(2)
+	if !ok {
+		panic("Couldn't recover runtime info")
+	}
+	fileName := fn[strings.LastIndex(fn, "/linkerd2/")+10:]
```

Comment:
TIOLI: move "/linkerd2/" into a const and use len of that const instead of magic number 10